CHANGELOG.md

```diff
@@ -0,0 +1,42 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [v2.1.0](https://github.com/awslabs/amazon-timestream-odbc-driver/releases/tag/v2.1.0) - 2024-07-10
+
+### Added
+
+- OLE DB support (#11).
```

Comment:
Can we add links for all of the items. The links will be repeating but it's useful to see which PR has the associated changes.

Reply:
I have added links to pull requests for all items. I have also added the item for when you fixed links in the documentation.